packages/core-components/src/components/CopyTextButton/CopyTextButton.tsx

```diff
@@ -49,6 +49,10 @@ export interface CopyTextButtonProps {
   tooltipText?: string;
 }
 
+type LabelledCopyTextButtonProps = CopyTextButtonProps & {
+  ariaLabel?: string;
```

Comment:
question here, maybe @rugvip too, maybe we should make this `aria-label` instead of `ariaLabel`. Thinking that we should keep with the traditional pattern for this.

Reply:
Noting that there's some usage of `ariaLabel` internally, but +1 to forwarding this as `aria-label` in the public API surface.